code/modules/reagents/reagent_containers/food/mobcube.dm

```diff
@@ -78,11 +78,15 @@
 		to_chat(M, "<span class='warning'>\The [src] expands!</span>")
 	if(!contained_mob)
 		return
+	var/mob/C = contained_mob
```

Comment:
I'd keep this a simple ``var/mob/C`` define here in order to avoid an unncessary use of memory. If ``contained_mob`` is a path, then writing it here is a waste of time since it's unused and almost immediately rewritten (and while technically allowed in DM, it's also assigning an invalid type to the variable. Bad practice)...

Reply:
done